src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -1591,6 +1591,15 @@ internal override void ForceComplete(SourceLocation? locationOpt, Predicate<Symb
                                 }
 
                                 DeclaringCompilation.SymbolDeclaredEvent(this);
+                                if (this.IsPartialDefinition())
```

Comment:
This check is similar in purpose to the following check performed for ordinary methods. Essentially it seems that MethodCompiler will report the SymbolDeclaredEvents for implementation parts of partial methods. But for the definition parts, the symbol itself must do it.

https://github.com/dotnet/roslyn/blob/eea02d9bef05717cb36f899d81a430a774a0605e/src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs#L279-L285